src/Microsoft.AspNetCore.SignalR.Core/DefaultHubLifetimeManager.cs

```diff
@@ -150,7 +150,7 @@ public override Task InvokeGroupExceptAsync(string groupName, string methodName,
 
         private InvocationMessage CreateInvocationMessage(string methodName, object[] args)
         {
-            return new InvocationMessage(GetInvocationId(), nonBlocking: true, target: methodName, argumentBindingException: null, arguments: args);
+            return new InvocationMessage(null, target: methodName, argumentBindingException: null, arguments: args);
```

Comment:
Why do you need to pass null? Also can you remove the GetInvocationId method?

Reply:
Just missed updating this. Though the above comment means we need to rethink the plan for this a little.